commit 9d458e89303a77c1accada85b36071b2880d37f7 (refs/changes/56/2356/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-08-09T08:15:40-07:00 (2 years, 2 months ago)
    
    joyent/node-triton#222 use `TRITON_ROLE` instead of `MANTA_ROLE` env var

diff --git a/CHANGES.md b/CHANGES.md
index 4d36359..5e3bb57 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -7,6 +7,9 @@ Known issues:
 
 ## not yet released
 
+- [joyent/node-triton#222] Fix the matching environment variable for the
+  `triton -r,--role ROLE ...` option to be `TRITON_ROLE` instead of
+  `MANTA_ROLE`.
 - [joyent/node-triton#201] Fix `triton -r,--role ROLE ...` option for taking up
   an RBAC role. This was introduced in v4.12.0 and was accidentally broken
   in v5.0.0.
diff --git a/lib/cli.js b/lib/cli.js
index ed9e41e..089da96 100644
--- a/lib/cli.js
+++ b/lib/cli.js
@@ -104,9 +104,9 @@ var OPTIONS = [
     {
         names: ['role', 'r'],
         type: 'arrayOfCommaSepString',
-        env: 'MANTA_ROLE',
+        env: 'TRITON_ROLE',
         help: 'Assume an RBAC role. Use multiple times or once with a list',
-        helpArg: 'ROLE,ROLE,...'
+        helpArg: 'ROLE,...'
     },
     {
         names: ['keyId', 'k'],
